packages/simulator/lib/elastic.js

```diff
@@ -0,0 +1,59 @@
+const elastic = require('elasticsearch')
```

Comment:
This import crashes on my machine, it needs to be '@elastic/elasticsearch'. How are our environments different? 😕